tes-adapter/src/main/java/com/epam/pipeline/tesadapter/service/TesTaskServiceImpl.java

```diff
@@ -46,4 +62,19 @@ private Long parseRunId(String id) {
             throw new IllegalArgumentException(e);
         }
     }
+
+    @Override
+    public TesServiceInfo getServiceInfo() {
+        Stream<TesServiceInfo> tesServiceInfoStream = Stream.of(new TesServiceInfo())
```

Comment:
I think that it is an over-engineering here, why you suppose that using streams here is a good way? Lets Create this object just with old fashion java:
````
TesServiceInfo tesServiceInfo = new TesServiceInfo();
tesServiceInfo.set....
tesServiceInfo.set....
tesServiceInfo.set....
return tesServiceInfo
````